packages/client/test/integration/NetworkNodeFacade.test.ts

```diff
@@ -77,7 +79,9 @@ describe('NetworkNodeFacade', () => {
                     privateKey: fastPrivateKey()
                 },
                 network: {
-                    id: nodeId,
+                    networkNode: {
```

Comment:
Could rename the `networkNode` field in `NetworkOptions` could be renamed to `node`. No need for prefix as all configuration in that options is network-related.